KIPs/kip-7-fungible_token.md

```diff
@@ -27,9 +27,12 @@ A standard interface allows any token on Klaytn to be re-used by other applicati
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Klaytn platforms (klaytn). -->
 This document derived heavily from Ethereum's [ERC-20 token standard](https://eips.ethereum.org/EIPS/eip-20) written by Fabian Vogelsteller and Vitalik Buterin.
-Comparing with ERC-20 token standard, this token standard has two differences:
-- includes three more optional functions (mint, burn, and burnFrom)
+Comparing with ERC-20 token standard, this token standard has some differences:
+- can include optional functions (e.g. mint, burn, pausable methods)
```

Comment:
```suggestion
- can include optional functions (e.g. mintable, burnable and pausable methods)
```
How about matching form?
"mintable, burnable and pausable methods" or "mint, burn and pause"